docs/using-in-frontend.md

```diff
@@ -0,0 +1,37 @@
+# Using nearley in browsers
+
+Use a tool like [Webpack](https://webpack.js.org/) or [Rollup](https://rollupjs.org/) to include the `nearley` NPM package in your browser code.
```

Comment:
Wait, doesn't nearley work fine if you just, y'know, include nearley.js?

Let's be very careful making the distinction between _parsing_ with nearley and _compiling_ with nearley**c**. This file should be called `compiling nearley grammars in browsers` or something!